startup/99-helper_functions.py

```diff
@@ -166,3 +184,29 @@ def wait_for_peaks(pool_interval, timeout, peaks, peaks_fields=None):
                 break
             else:
                 yield from bps.sleep(pool_interval)
+
+def _block_beam(block_beam_bit): ##TODO move this to a child of inout
+"""Protective inout function that only moves if necessary. The EPS will always actuate on bps.mv.
+
+Parameters:
+------------
+block_beam_bit : int, bool
+"""
+    if block_beam_bit: 
```

Comment:
```suggestion
    """Protective inout function that only moves if necessary.
    The EPS will always actuate on bps.mv.

    Parameters:
    ------------
    block_beam_bit : int, bool
    """
    if block_beam_bit: 
```

Reply:
gosh, you did it before I could make the time.  But now I don't have to feel bad about pulling this at CSX and practicing rebase